parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -177,6 +178,17 @@ downloadContainerd() {
     retrycmd_get_tarball 120 5 "$CONTAINERD_DOWNLOADS_DIR/${CONTAINERD_TGZ_TMP}" ${CONTAINERD_DOWNLOAD_URL} || exit $ERR_CONTAINERD_DOWNLOAD_TIMEOUT
 }
 
+installAPMZ() {
+  apmz_filepath=/usr/local/bin/apmz; apmz_url="https://github.com/devigned/apmz/releases/download/v0.0.1/apmz_v0.0.1_linux_amd64.tar.gz"
```

Comment:
why is this download path in your fork?

Reply:
Well... I wrote `apmz`. I'm happy to move it to Azure if we like what's going on with it :).

Reply:
aaah nvm :)